modules/manage/pages/cluster-maintenance/compaction-settings.adoc

```diff
@@ -0,0 +1,47 @@
+= Compaction Settings
+:description: Redpanda's approach to compaction and options for configuring it.
+
+Configure compaction for your cluster to optimize storage utilization.
+
+== Redpanda compaction overview
+
+Compaction is an optional mechanism intended to reduce the storage needs of Redpanda topics. You can enable compaction through configuration of a cluster or topic's cleanup policy. When compaction is enabled as part of the cleanup policy, a background process executes on a pre-set interval to perform compaction operations. When triggered for a partition, the process purges older versions of messages for a given key and only retains the most recent message in that partition. This is done by analyzing closed segments in the partition, copying the most recent messages for each key into a new segment, then deleting the source segments.
+
+image::shared:compaction-example.png[Example of topic compaction]
+
+This diagram provides an illustration of a compacted topic. Imagine a remote sensor network that uses image recognition to track appearances of red pandas in a geographic area. The sensor network employs special devices that send messages to a topic when they detect one. You might enable compaction to reduce topic storage while still maintaining a record in the topic of the last time each device saw a red panda, perhaps to see if they stop frequenting a given area. The left side of the diagram shows all messages sent across the topic. The right side illustrates the results of compaction; older messages for certain keys are deleted from the message log.
+
+IMPORTANT:  When using Tiered Storage, compaction functions at the local storage level before segments are uploaded to storage. Once a segment is uploaded to storage it is not retrieved for further compaction operations. A key may therefore appear in multiple segments between Tiered Storage and local storage.
+
+While compaction reduces storage needs, Redpanda's compaction (just like Kafka's) does not guarantee perfect de-duplication of a topic. Duplicates of a message, or additional messages, may still exist within a topic. Compaction is not a complete topic operation; rather, it represents a best effort mechanism to reduce storage needs for topics created within a certain time window.
```

Comment:
I kept the discussion here higher level, focusing on topics for simplicity and to mimic Kafka's own documentation on compaction. This was also because there are scenarios where a key can begin saving to a different partition. The language you called out was definitely in error. I simplified it as follows:

While compaction reduces storage needs, Redpanda's compaction (just like Kafka's) does not guarantee perfect de-duplication of a topic. It represents a best effort mechanism to reduce storage needs but duplicates of a key may still exist within a topic. Compaction is not a complete topic operation, either, it operates on subsets of each partition within the topic.